build/localnet/cluster-manager/main.go

```diff
@@ -24,6 +24,12 @@ const cliPath = "/usr/local/bin/client"
 var (
 	rpcURL string
 	logger = pocketLogger.Global.CreateLoggerForModule("cluster-manager")
+
+	autoStakeAmount = "150000000001"
+	autoStakeChains = []string{"0001"}
+	// autoStakeSkipStakeForValidatorIds is a list of validator ids that should not be auto-staked
+	// it is used to avoid auto-staking the validators that are already staked as part of genesis.
+	autoStakeSkipStakeForValidatorIds = []string{"001", "002", "003", "004"}
```

Comment:
I made it configurable that way so that the developer can easily change the behaviour by modifying that line thanks to hot-reload.

The fact that by default is coupled to genesis.json was kinda by design in my head.

I didn't want to get too smart because it's easy to fall into the rabbit hole and make something hard to understand/manage.

What I wanted was a "sane default" that provided a better experience, for more advanced scenarios, like the one you describe here (of course totally valid), I would approach the problem like:

"Let's have some people play with it ➡️ collect feedback ➡️  improve ➡️  repeat."

Wdyt?

Reply:
Fair enough.

If/when it breaks, gives me an opportunity to play around with it myself :)